SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMigration/SpatialTestCharacterMigration.cpp

```diff
@@ -73,10 +73,9 @@ void ASpatialTestCharacterMigration::PrepareTest()
 
 			FActorSpawnParameters SpawnParams;
 			SpawnParams.Owner = PlayerController;
-			AActor* TestActor = GetWorld()->SpawnActor<AActor>(AActor::StaticClass(), FTransform(), SpawnParams);
+			AActor* TestActor = SpawnActor<AActor>(SpawnParams);
```

Comment:
Presumably this does the same thing? Not sure why we were using AActor::StaticClass() before

Reply:
ah right, makes sense